src/roles/vsd-postdeploy/tasks/main.yml

```diff
@@ -13,3 +13,11 @@
     postdeploy_only: True
     skip_vsc: True
     check_vsd_license: "{{ vsd_license_file is defined }}"
+
+- name: Executing VSD inplace upgrade if the vsd_install_upgrade is true
```

Comment:
Generally, we don't allow post-deploy to make changes to the system. Can you move this to vsd-deploy?

Reply:
done